schema/dataKeys.json

```diff
@@ -340,6 +348,21 @@
                   "type": "string"
                 }
               },
+              "source": {
```

Comment:
This is from the snapshot, it shouldn't be in the mapping. e2e is wrong

Reply:
removed it